runner/options.go

```diff
@@ -536,7 +535,9 @@ func (options *Options) ValidateOptions() error {
 		gologger.Debug().Msgf("Store response directory specified, enabling \"sr\" flag automatically\n")
 		options.StoreResponse = true
 	}
-
+	if options.StoreResponseDir != "" {
+		os.RemoveAll(path.Join(options.StoreResponseDir, "index.txt"))
```

Comment:
`path` is for **URLs only**. 
Files requires `filepath` to be cross platform

Reply:
Done